resources/fulfillment.js

```diff
@@ -82,6 +99,23 @@ Fulfillment.prototype.createV2 = function createV2(params) {
   return this.shopify.request(url, 'POST', this.key, params);
 };
 
+/**
+ * Lists fulfillments for a fulfillment order.
+ *
+ * @param {Number} fulfillmentOrderId Fulfillment order ID
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Fulfillment.prototype.listV2 = function listV2(fulfillmentOrderId) {
```

Comment:
Alternatively, this could be `fulfillmentOrder.fulfillments` instead of `fulfillment.listV2` – which is better?

Reply:
It should be on `FulfillmentOrder` but this https://shopify.dev/api/admin-rest/2022-10/resources/fulfillmentorder#get-orders-order-id-fulfillment-orders already exists as the current `fulfillmentOrder.list()` so I'm not sure if this is needed.